src/crimson/os/seastore/btree/fixed_kv_node.h

```diff
@@ -998,7 +999,9 @@ struct FixedKVLeafNode
   // 2. The child extent is stable
   bool is_child_stable(uint16_t pos) const final {
     auto child = this->children[pos];
-    if (is_valid_child_ptr(child)) {
+    if (is_reserved_ptr(child)) {
+      return true;
```

Comment:
Seems to me `is_child_stable()` is not useful/meaningful for the reserved case.

If that's reasonable to you, can we add comments to both `is_child_stable(pos)` here and `PhysicalNodeMapping::is_stable()` that the return value is undefined (but won't crash) if the child/pin is reserved ?

Reply:
> Seems to me `is_child_stable()` is not useful/meaningful for the reserved case.

Yeah, I just wanted to avoid crash

> 
> If that's reasonable to you, can we add comments to both `is_child_stable(pos)` here and `PhysicalNodeMapping::is_stable()` that the return value is undefined (but won't crash) if the child/pin is reserved ?

Will do.